src/utilities/assert.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Why are we adding this file?  What is the problem with the assert from <cassert>?

Reply:
Don't need it. I was going to ask about replacing the assert macros with this regarding the comment in src/zmqlib/zmqLibrary/zmqHelper.cpp (NEED to make this a throw operation instead once exceptions are integrated)

And then I accidentally added src/utilities/ instead of the files individually, so I fixed it rather than taking it out. I can take it out if you want

Reply:
interesting, I will look at it a little more.